README.md

```diff
@@ -84,10 +86,11 @@ global:
   num_classes: 2                    # Number of classes
   data_path: /path/to/data/folder   # Path to folder containing samples
   number_of_bands: 3                # Number of bands in input images
-  model_name: unetsmall				# One of unet, unetsmall, checkpointed_unet, ternausnet, or inception
+  model_name: unetsmall             # One of unet, unetsmall, checkpointed_unet, ternausnet, fcn_resnet101, deeplabv3_resnet101 or inception
   bucket_name:                      # Name of the S3 bucket where data is stored. Leave blank if using local files
   task: segmentation                # Task to perform. Either segmentation or classification.
   num_gpus: 0                       # Number of GPU device(s) to use. Default: 0
+  debug_mode: True                  # if True, displays useful info for debugging
 
```

Comment:
scale_data: [0, 1]                # Min and Max for input data rescaling. Default: [0, 1]. Enter False if no rescaling is desired.